tests/app/notifications/rest/test_send_notification.py

```diff
@@ -1219,3 +1220,34 @@ def test_should_throw_exception_if_notification_type_is_invalid(client, sample_s
         headers=[('Content-Type', 'application/json'), auth_header])
     assert response.status_code == 400
     assert json.loads(response.get_data(as_text=True))["message"] == err_msg
+
+
+@pytest.mark.parametrize("notification_type, recipient",
+                         [("sms", '07700 900 855'),
+                          ("email", "test@gov.uk")
+                          ]
+                         )
+def test_post_notification_should_set_reply_to_text(client, notify_db, notify_db_session, mocker, notification_type,
+                                                    recipient):
+    mocker.patch('app.celery.provider_tasks.deliver_{}.apply_async'.format(notification_type))
+    service = create_service()
+    template = create_sample_template(notify_db=notify_db, notify_db_session=notify_db_session,
+                                      service=service, template_type=notification_type)
+    expected_reply_to = current_app.config['FROM_NUMBER']
+    if notification_type == EMAIL_TYPE:
```

Comment:
If its parametrised should there be  SMS block too?

Reply:
The default is SMS